.github/workflows/pr-title.yml

```diff
@@ -38,5 +40,7 @@ jobs:
             snowflake
             sqlite
             types
```

Comment:
I propose to remove `types`, as it doesn't really make sense anymore with the monorepo

Reply:
Done